include/small/lsregion.h

```diff
@@ -274,6 +274,9 @@ lsregion_aligned_alloc(struct lsregion *lsregion, size_t size, size_t alignment,
 	return res;
 }
 
+#define lsregion_alloc_object(lsregion, id, T)					\
+	(T *)lsregion_aligned_alloc((lsregion), sizeof(T), alignof(T), (id))
+
```

Comment:
Please add a test to ensure there's no typo in this macro.

Reply:
Done.